jetty-documentation/src/main/asciidoc/distribution-guide/annotations/using-annotations.adoc

```diff
@@ -121,9 +121,8 @@ As is the case with annotation scanning, the link:#using-extra-classpath-method[
 
 ____
 [NOTE]
-As of Jetty-9.4.4, unless the `web.xml` is version 3.0 or greater, only `ServletContainerInitializers` that are on the container classpath will be discovered.
-Users wishing to use `ServletContainerInitializers` from within the webapp with older versions of `web.xml` must either upgrade their `web.xml` version, or call `WebAppContext.setConfigurationDiscovered(true)` either programmatically or in xml.
-Upgrading the `web.xml` version is preferable.
+As of Jetty 10, Annotations will be discovered even for old versions of `web.xml` (2.5).
+Users wishing to not use `ServletContainerInitializers` from within the webapp with older versions of `web.xml` must call `WebAppContext.setConfigurationDiscovered(false)` either programmatically or in xml.
```

Comment:
Actually it's not just SCIs, it is also @WebServlet, @WebFilter, @WebListener @MultiPart etc etc. So maybe phrase this as:
"Users wishing not to use annotations from the webapp classpath must call ...."